.azure-pipelines/sign-binary.yaml

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger: none
+
+pool:
+  name: Azure Pipelines
+  vmImage: windows-latest
+
+variables:
+  - name: ProductBinPath
+    value: '$(System.DefaultWorkingDirectory)\assets'
+  - name: sourceRepository
+    value: 'microsoftgraph/msgraph-cli'
+  - name: repositoryConnection
+    value: 'GitHub - calebkiage'
+  - name: zipTemplate
+    value: msgraph-cli-win-x64-{0}.zip
+parameters:
+  - name: tag
+    displayName: Tag
+    type: string
+    default: latest
+
+
+# download -> scan -> zip -> upload
+#          -> sign
+#
+
+stages:
+- stage: download
+  jobs:
+    - job: download
+      steps:
+      - checkout: none
+      # Download GitHub Release
+      # Downloads a GitHub Release from a repository
+      - task: DownloadGitHubRelease@0
+        inputs:
```

Comment:
switch to unbuntu as well for this one?

Reply:
Does ubuntu provide benefits over windows? 🤔